addons/point_of_sale/static/tests/tours/helpers/PartnerListScreenTourMethods.js

```diff
@@ -24,6 +39,15 @@ odoo.define('point_of_sale.tour.PartnerListScreenTourMethods', function (require
                 },
             ];
         }
+        settleButtonTextIs(name) {
+            return [
+                {
+                    content: 'check the content of the settle button',
+                    trigger: `.button.settle-due:contains(${name})`,
+                    run: () => {},
+                },
+            ];
+        }
```

Comment:
given that the `settle` button is in enterprise, is this method useful in community ?